### ISSUE TITLE
Download Permission

### ISSUE THREAD

**Issue by RichPruitt** [roman-yagodin/R7.Documents#61]
Would be nice to have a separate permission to be able to download.

One permission to view the documents, another permission to download.

**Comment by roman-yagodin** [comment]
You mean permission to view just that a document exists?

**Comment by RichPruitt** [comment]
Yes, a permission to view the module and another permission to download a file in the module.

Currently I have to hide the whole module if you do not have download permissions, and I have another html/text module that says 'no permissions to download any files'.
